libs/SalesforceSDK/build.gradle

```diff
@@ -17,6 +17,7 @@ dependencies {
     api project(':libs:SalesforceAnalytics')
     api 'com.squareup.okhttp3:okhttp:3.9.0'
     api 'com.google.android.gms:play-services-gcm:12.0.1'
+    api 'com.android.support:support-compat:26.1.0'
```

Comment:
Adding the app compat library. Lotsa useful stuff in here, including `JobIntentService`.